seed/php-sdk/pagination/property-accessors/src/dynamic-snippets/example5/snippet.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Example;
+
+use Seed\SeedClient;
+use Seed\Users\Requests\ListUsersCursorPaginationRequest;
+use Seed\Users\Types\Order;
+
+$client = new SeedClient(
+    token: '<token>',
+    options: [
+        'baseUrl' => 'https://api.fern.com',
+    ],
+);
+$client->users->listWithCursorPagination(
+    new ListUsersCursorPaginationRequest([
+        'page' => 1.1,
+        'perPage' => 1.1,
```

Comment:
These might still be failing (due to an example generation bug that needs to be resolved). It's ok if this is added to `allowedFailures` for now.